src/views/AnnualReport.vue

```diff
@@ -29,6 +29,12 @@
       attach="#annual-report"
     />
 
+    <bcol-error-dialog
+      :bcolOBject="bcolObj"
+      filingType="Annual Report"
```

Comment:
You're not actually passing a "type" here; you're passing a name. There's a mixin that can help you clean this up! Search for `filingTypeToName` method.

Reply:
I thought something like this existed, couldn't track it down...thanks!

Reply:
Fixed.